docs/installation_configuration_guide/Common.md

```diff
@@ -19,11 +19,15 @@ Nussknacker configuration is divided into several configuration areas, each area
 
 * [Designer](/about/GLOSSARY#nussknacker-designer) configuration (web application ports, security, various UI settings, database),
 * Scenario Types configuration, comprising of:
-  * [Deployment Manager](/about/GLOSSARY#deployment-manager) configuration, 
   * [Model](/about/GLOSSARY#model) configuration.
-  * [Category](/installation_configuration_guide/DesignerConfiguration/#scenario-type-categories) configuration
+  * [Deployment](/about/GLOSSARY#deployment-manager) configuration,
+  * [Category](./DesignerConfiguration.md/#scenario-type-categories) configuration
 
-The Scenario Type is a convenient umbrella term for a particular Deployment Manager configuration and the associated model configuration. Diagram below presents main relationships between configuration areas.
+[Model](/about/GLOSSARY#model) configuration defines which components and which [Processing Mode](/about/ProcessingModes) will be available for the user. 
+[Deployment](/about/GLOSSARY#deployment-manager) configuration defines how scenario using these components will be deployed on the [Engine](/about/engine).
```

Comment:
The change from `Deployment manager configuration` to `Deployment configuration` is a bit problematic for me:

1. It may introduce some confusion when looking at the graph - We have 3 entities where we use word `deployment` - `Nussknacker deployment`, `Deployment manager` and `deploymentConfig`.

For this I suggest stating clearly that Deployment configuration = Deployment Manager configuration (although I'm not sure if that's not going to lead to some confusion with things like `configExecutionOverrides` on lite engine).

2. From the graph it seems to me we are configuring "The entity that manages the deployments" - here we are referring to it in a way that suggests we are configuring the deployments themselves `Deployment configuration`. Later in the configuration section we are referring to it as "Configuration of DeploymentManager". On the other hand the Deployment Manager configuration section is just `Deployment` on the sidebar. 

For this I think we should make some changes to the graph, but this may be a bigger conversation.

Reply:
I replaced everywhere to `Scenario deployment` instead of `Deployment Manager`. Also changed `Nussknacker deployment` into `Nussknacker Designer`. Also removed architecture related things from the graph.